### ISSUE TITLE
Is this project still being maintained? 

### ISSUE THREAD

**Issue by shams-ahmed** [nickoneill/PermissionScope#232]
Hey @nickoneill ,

As the title say is this project still being maintained? I can see there quite a few issues for feature/improvements and PR... 

The last commit to master was a few months ago...

**Comment by nickoneill** [comment]
No, I haven't had enough time to give this the attention it deserves and there are quite a few improvements to be made. Happy to accept the help if someone wants to organize a bit and plan to bring some PRs in.

**Comment by gaming-hacker** [comment]
has anyone have a list of localizations available?

**Comment by rlaferla** [comment]
Is there an alternative project to replace PermissionScope?
